### PR TITLE
Use deb repo

### PR DESCRIPTION
**Pull request by niklassc-xyz**

### CODE REVIEW

**Comment by niklassc-xyz** [comment]
I thought docker-compose wasn't in the stretch repos, that's why I at first didn't use apt.

I removed `cache_valid_time: 1800` because it led to an error when the cache was refreshed within the last 30 minutes. In this pull request, however, its back.

File: defaults/main.yml [hunk 1]
```diff
@@ -1,2 +1,5 @@
 ---
 docker_apt_repo: deb [arch=amd64] https://download.docker.com/linux/debian {{ ansible_distribution_release }} stable
+install_docker_compose: False
```
Comment (uubk) [inline defaults/main.yml]: Please prefix with `docker_` (e.g. `docker_install_compose`)

File: defaults/main.yml [hunk 2]
```diff
@@ -1,2 +1,5 @@
...
+docker_compose_version: "1.25.0"
```
Comment (uubk) [inline defaults/main.yml]: This is no longer used, right?

File: tasks/main.yml [hunk 3]
```diff
@@ -42,3 +42,13 @@
     state: present
     update_cache: yes
     cache_valid_time: 1800
+
+- name: Install docker-compose
+  when: install_docker_compose | bool
```
Comment (uubk) [inline tasks/main.yml]: The `|bool` filter should not be necessary, as the variable is already a boolean.

**Review by uubk — CHANGES_REQUESTED** [review]
A few small nitpicks, otherwise this looks good :+1:

**Comment by uubk** [comment]
Rebased and merged manually (which GitHub can't detect), closing PR.

**Pull request closed without merging**